app/manual/06_project_management.md

```diff
@@ -213,6 +214,17 @@ This entry tells NAP to look for the `freetype.dll` (on which *napfont* depends)
 
 *Note that in a future version of NAP the `WindowsDllSearchPaths` property will be deprecated and replaced by the `Windows` element of [LibrarySearchPaths](@ref user_module_config_search_paths)*
 
+### Data Search Paths {#user_module_config_data_search_paths}
+This entry defines paths to search for module data in, use these locations to store for example hardcoded shader files of your module.
+
+```
+"DataSearchPaths": [
+	"{PROJECT_DIR}/module/data"
+]
+```
+Use the [findAsset](https://docs.nap.tech/d9/d25/classnap_1_1_module.html#aedd8e1de61bda74c9bf952f492ddccd7) function of the Module class to retrieve paths e.g.: `mYourModuleService->getModule()->findAsset('someshader.frag')` and then read them using a function such as [nap::utility:readFileToString](https://docs.nap.tech/dd/db0/namespacenap_1_1utility.html#a2a2dc66faeb526bab38cfb2b7858018d)
```

Comment:
It's better not to link to the generated docs, instead let `doxygen` resolve the link during generation. This is safer because the link to the page can change from generation to generation (there's no guarantee).

Please change it to:

```
[nap::utility:readFileToString](@ref nap::utility::readFileToString)
```